independent-projects/junit5-virtual-threads/pom.xml

```diff
@@ -56,6 +56,11 @@
             <scope>compile</scope>
             <version>${junit.jupiter.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.25.3</version>
```

Comment:
Can you extract it as a variable? Also, does it needs to be in the test scope?

Reply:
Fixed. No idea how I could miss the scope :)